Source/BugsnagNotifier.m

```diff
@@ -0,0 +1,50 @@
+//
+//  BugsnagNotifier.m
+//  Bugsnag
+//
+//  Created by Jamie Lynch on 29/04/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+#import "BugsnagNotifier.h"
+
+@implementation BugsnagNotifier
+
+- (instancetype)init {
+    if (self = [super init]) {
+
+        self.name =
+#if TARGET_OS_TV
+                @"tvOS Bugsnag Notifier";
+#elif TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE
+                @"iOS Bugsnag Notifier";
+#elif TARGET_OS_MAC
+        @"OSX Bugsnag Notifier";
+#else
+            @"Bugsnag Objective-C";
+#endif
+        self.version = @"5.23.0";
```

Comment:
We need to ensure the Makefile works to set the version (it appears to be broken on v6 as it looks for `NOTIFIER_VERSION` in this file (from it's previous purpose as the client-like class).

Suggest you move `NOTIFIER_VERSION` and `NOTIFIER_URL` to this file?

Reply:
Updated the Makefile to alter the version on this line instead. I've removed the unused `NOTIFIER_VERSION` and `NOTIFIER_URL` constants.